library/registry

```diff
@@ -1,12 +1,12 @@
-# maintainer: Joffrey F <joffrey@docker.com> (@shin-)
-# maintainer: Sam Alba <sam@dotcloud.com> (@samalba)
 # maintainer: Stephen Day <stephen.day@docker.com> (@stevvooe)
 # maintainer: Olivier Gambier <olivier.gambier@docker.com> (@dmp42)
+# maintainer: Richard Scothern <richard.scothern@gmail.com>
 
 latest: git://github.com/docker/docker-registry@0.9.1
 0.8.1: git://github.com/docker/docker-registry@0.8.1
 0.9.1: git://github.com/docker/docker-registry@0.9.1
 
-2: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
-2.0: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
+2: git://github.com/docker/distribution@1341222284b3a6b4e77fb64571ad423ed58b0d34
+2.0: git://github.com/docker/distribution@1341222284b3a6b4e77fb64571ad423ed58b0d34
 2.0.0: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
```

Comment:
@tianon It's standard procedure to not get rid of old releases when releasing new software. For example, if someone setup there infrastructure to pin to that version, we should not break it by removing that version. No, it is not supported, but if they want to run that version, it is their choice and not ours.